llvm-10.0.1/llvm/lib/Target/X86/X86InstrInfo.cpp

```diff
@@ -674,9 +674,9 @@ X86InstrInfo::getDestAndSrc(const MachineInstr &MI) const {
       case X86::CMOV16rr:
       case X86::CMOV32rr:
       case X86::CMOV64rr: {
-      // These are conditional moves.
-      // What to do??
-      return None;
+        /* FIXME: */
```

Comment:
What is the purpose of creating these "empty DestSourcePairs"?

Reply:
CMOV has a destination operand and needs to be handled. Currently we return None and ignore it. There were two tests that failed due to the use of CMOV64rr. I intend to handle CMOV in the next commit. MIR that has no destination operand do not affect the taint propagation (like CMP). So I've used it as a way to by pass the check.